automatic/freecad/update_helper.ps1

```diff
@@ -23,13 +24,16 @@ param(
       $veri = ((($url64 -split('\/'))[-1]) -replace( "(x\d{2})|(_\d{2}\-py\d{2})|(\-)?([A-z])+?(\-)|(\.$ext)", ''))
       "veri -$veri-" | Write-Warning
       $DevRevision,$year,$month,$day = (($veri -replace('\-','.') ) -split('\.'))
-      [version]$DevVersion = (( Get-Content "$ScriptLocation\freecad.json" | ConvertFrom-Json ).dev) -replace("-$kind",'')
-      # Catch to make sure version will be the latest version
-      if (( $DevVersion.Minor -lt ($year.Substring(0,2)) ) -and ( $DevVersion.Build -gt $filler )) { [version]$DevVersion = "0.20.${filler}" }
-      if (($DevRevision -match "\d{5}") -and ($DevRevision -le "29192")){
-        "Revision is less than it was on date 6/20/2022 which could mean a new version minor" | Write-Warning
-        "Going to increase the version minor by 1" | Write-Warning
-        [version]$version = ( ( ($DevVersion.Major),(($DevVersion.Minor + 1)),($DevVersion.Build),($DevRevision) ) -join "." )
+      [version]$DevVersion = (( Get-Content "$ScriptLocation\freecad.json" | ConvertFrom-Json ).stable)
+      if (($DevRevision -match "\d{5}") -and ($DevRevision -gt "29192")){
```

Comment:
could you help me understand what this specific revision number represents and why you need the check?

Reply:
This revision number was the first number updated that I found using minor 21

Reply:
but I still don't get why the check, in what situation would it not be true?